datafusion/common/src/config.rs

```diff
@@ -314,6 +314,85 @@ config_namespace! {
     }
 }
 
+/// When using the parquet feature,
+/// use the same default writer settings as the extern parquet.
+#[cfg(feature = "parquet")]
```

Comment:
I think the idea of defining these constants is to try and keep the values in sync with what is in the arrow-rs writer. 

However, since we now have a test that verifies the values it is my opinion that this additional level of indirection (and the stubs required with a different feature flag) make the code harder to read for minimal additional benefit. 

In other words, I think keeping the config values as hard coded constants with the tests is sufficient and we could remove the `parquet_defaults` modules